src/Theme/index.js

```diff
@@ -85,6 +95,33 @@ const TextWrapper = styled(Text)`
   color: ${({ color, theme }) => theme[color]};
 `;
 
+export const Typography = {
+  custom: ({ color, ...rest }) => <TextWrapper color={color || 'text1'} {...rest} />,
```

Comment:
```suggestion
  custom: ({ color, sx }) => <TextWrapper color={color || 'text1'}  sx={sx} />,
```

Reply:
We can remove the `..rest` but then we have to explicitly pass the `children` property and render it inside the `TextWrapper`:

```js
custom: ({ color, children, sx }) => (
  <TextWrapper color={color || 'text1'} sx={sx}>
    {children}
  </TextWrapper>
),
```

I guess it's still ok :)